tests/baselines/reference/tsxElementResolution19.symbols

```diff
@@ -0,0 +1,28 @@
+=== tests/cases/conformance/jsx/react.d.ts ===
+
+No type information for this code.declare module "react" {
+No type information for this code.
+No type information for this code.}
+No type information for this code.
+No type information for this code.=== tests/cases/conformance/jsx/file1.tsx ===

```

Comment:
Seems to be in a lot of places: https://github.com/Microsoft/TypeScript/search?utf8=%E2%9C%93&q=No+type+information+for+this+code&type=Code